llvm/lib/Target/AMDGPU/SIFoldOperands.cpp

```diff
@@ -781,6 +781,27 @@ bool SIFoldOperandsImpl::isUseSafeToFold(const MachineInstr &MI,
   return !TII->isSDWA(MI);
 }
 
+static MachineOperand *lookUpCopyChain(const SIInstrInfo &TII,
```

Comment:
It does more than adding test. Needs restacking?

Reply:
yes